Documentation/op-guide/etcd2.rules

```diff
@@ -0,0 +1,121 @@
+### General cluster availability ###
+
+# alert if another failed peer will result in an unavailable cluster
+ALERT InsufficientPeers
+  IF count(up{job="etcd-k8s"} == 0) > (count(up{job="etcd-k8s"}) / 2 - 1)
```

Comment:
what is etcd-k8s? we need to define the job first. also i would like to remove the k8s mentioning in this doc.